core/src/main/thrift/client.thrift

```diff
@@ -331,6 +338,18 @@ service ClientService {
     1:ThriftTableOperationException tope
   )
 
+  string executeAdminOperation(
+      3:trace.TInfo tinfo
+      4:security.TCredentials credentials
+      2:AdminOperation op
+      5:list<string> arguments
+      6:set<i64> filtertxids
+      7:list<string> filterStatues
+      8:string secretOption
```

Comment:
For a new method that doesn't need to preserve backwards compatibility, the parameters can be numbered in order starting at 1.